frontend/packages/network-attachment-definition-plugin/src/constants/constants.tsx

```diff
@@ -101,7 +126,8 @@ export type NetworkTypeParameter = {
   name: string;
   required?: boolean;
   type: string;
-  hintText?: string;
+  hintText?: React.ReactNode;
```

Comment:
Please drop the `React.` here.

Reply:
can't as we are missing the ts configuration in this plugin under console